app/tests/unit/containers/Forms/NaicsField.test.tsx

```diff
@@ -0,0 +1,66 @@
+import {NaicsFieldComponent} from 'containers/Forms/NaicsField';
+import {mount} from 'enzyme';
+import React from 'react';
+import {NaicsField_query} from '__generated__/NaicsField_query.graphql';
+import {createDefaultJsonSchemaFormProps} from 'tests/json-schema-utils';
+
+describe('The Naics Field custom field', () => {
+  const testQuery: NaicsField_query = {
+    ' $refType': 'NaicsField_query',
+    naicsCodes: {
+      edges: [
+        {
+          node: {
+            rowId: 123,
+            naicsCode: '111111',
+            naicsDescription: 'Javascript',
+            ciipSector: 'Software'
+          }
+        },
+        {
+          node: {
+            rowId: 124,
+            naicsCode: '222222',
+            naicsDescription: 'Typescript',
+            ciipSector: 'Software'
+          }
+        }
+      ]
+    }
+  };
+
+  const initialProps = {
+    ...createDefaultJsonSchemaFormProps(),
+    query: testQuery
+  };
+
+  it('Displays the list of returned NAICS codes as a dropdown', () => {
+    const props = {
+      ...initialProps,
+      formData: ''
+    };
+
+    const componentUnderTest = mount(<NaicsFieldComponent {...props} />);
+    expect(componentUnderTest).toMatchSnapshot();
+  });
+
+  it('Displays (<code> - <industry> - <description>) when the form data contains a 6-digit NAICS code', () => {
```

Comment:
```suggestion
  it('Displays (<code>  - <description>) when the form data contains a 6-digit NAICS code', () => {
```
Update when removing the industry